openfl/utilities/utils.py

```diff
@@ -236,3 +236,22 @@ def merge_configs(
             settings.set(key, operation(value))
     settings.validators.validate()
     return settings
+
+
+def change_tags(tags, add_field=None, remove_field=None) -> Tuple[str]:
```

Comment:
```suggestion
def change_tags(tags, *, add_field=None, remove_field=None) -> Tuple[str]:
```
Looks like we expect that we shouldn't use add_field and remove_field as positional arguments.